kerl

```diff
@@ -24,7 +24,7 @@
 
 unset ERL_TOP
 
-KERL_VERSION="1.3.4"
+KERL_VERSION="1.4.1"
```

Comment:
Also let's not forget to bump this to `1.4.2` before creating the next release :)